WinImageBuilder.psm1

```diff
@@ -764,6 +764,21 @@ function Run-Sysprep {
     Remove-VM $Name -Confirm:$false -Force
 }
 
+function Get-ImageInformation {
+    Param(
+        [Parameter(Mandatory=$true)]
+        [string]$driveLetter
+    )
+    $versionString = (Get-Item "$driveLetter\Windows\system32\ntdll.dll").VersionInfo.ProductVersion
+    $OSVersion = $versionString.split('.')
+    $ImageVersion = @{"Major"=$OSVersion[0];"Minor"=$OSVersion[1]}
+    $ImageArchitecture="AMD64"

```

Comment:
put a note here, to do

# To do (ilenghel): find a method to retrieve installation type

You can retrieve the architecture by checking if you have a "C:\Windows\SysWOW64"